### ISSUE TITLE
GTF parser fails if there is only one transcript in the file.

### ISSUE THREAD

**Issue by diekhans** [BrooksLabUCSC/flair#388]
Either of the attached files creates the error 
```

Traceback (most recent call last):
  File "/Users/markd/compbio/flair/projs/src/flair-markd/./bin/gtf_to_bed", line 148, in <module>
    main()
  File "/Users/markd/compbio/flair/projs/src/flair-markd/./bin/gtf_to_bed", line 23, in main
    gtf_to_bed(args.psl, args.gtf, isbed, args.include_gene, chrom_sizes=args.chrom_sizes)
  File "/Users/markd/compbio/flair/projs/src/flair-markd/./bin/gtf_to_bed", line 115, in gtf_to_bed
    if blockcount > 1 and blockstarts[0] > blockstarts[1]:  # need to reverse exons
       ^^^^^^^^^^
UnboundLocalError: cannot access local variable 'blockcount' where it is not associated with a value

```

Code also makes the invalid assumption that transcripts are ordered.  While this is almost always the case, it is not required, as far as I can tell.

[flair-386-1.gtf.gz](https://github.com/user-attachments/files/18032172/flair-386-1.gtf.gz)
[flair-386-2.gtf.gz](https://github.com/user-attachments/files/18032173/flair-386-2.gtf.gz)

**Comment by diekhans** [comment]
same as #386

**Comment by diekhans** [comment]
fixed by merge #389
